notifier/tasks.py

```diff
@@ -122,7 +128,26 @@ def batch_digest_subscribers():
 
     from_dt, to_dt = _time_slice(settings.FORUM_DIGEST_TASK_INTERVAL)
 
-    logger.info("Beginning forums digest task: from_dt=%s to_dt=%s", from_dt, to_dt)
+    # Remove old tasks from the database so that the table doesn't keep growing forever.
+    ForumDigestTask.prune_old_tasks(settings.FORUM_DIGEST_TASK_GC_DAYS)
+
+    if self.request.retries == 0:
+        task, created = ForumDigestTask.objects.get_or_create(
+            from_dt=from_dt,
+            to_dt=to_dt,
+            defaults={'node': platform.node()}
+        )
+        if created:
+            logger.info("Beginning forums digest task: from_dt=%s to_dt=%s", from_dt, to_dt)
+        else:
```

Comment:
Yes, the intervals are calculated by the (`_time_slice`)[https://github.com/open-craft/notifier/blob/a43141a4c70813ce13d368a1f40f4776193a9b00/notifier/tasks.py#L67-L110] function and are identical given identical `FORUM_DIGEST_TASK_INTERVAL` setting.